src/odemis/driver/test/zeiss_test.py

```diff
@@ -238,12 +238,12 @@ def test_move_axes(self):
         tstart = time.time()
         self.stage.moveRelSync({"x": 1e-3})
         dur = time.time() - tstart
-        self.assertGreaterEqual(dur, 1.1, "1 mm move took only %g s" % dur)
+        self.assertGreaterEqual(dur, 1.01, "1 mm move took only %g s" % dur)
```

Comment:
Could we show how we got to 1.01, if there is a formula ? In that way if there are any changes to the simulator, these cases can be independent.

Reply:
Well... it's not great. The simulator goes at 2mm/s . This requests a move of 1mm, so it should take 0.5s. But, the driver wait always at least 1s. So we are stuck to 1s (or a tiny bit more) unless the move is > 2mm. Maybe we should move by 3 mm....

Reply:
I've now changed the test case to move by 4mm. So it should take 2s. I've also added a comment explaining how to compute this value.